csharp/ql/src/Security Features/CWE-022/ZipSlip.ql

```diff
@@ -1,90 +1,20 @@
 /**
- * @name Potential ZipSlip vulnerability 
- * @description When extracting files from an archive, don't add archive item's path to the target file system path. Archive path can be relative and can lead to 
- * file system access outside of the expected file system target path, leading to malicious config changes and remote code execution via lay-and-wait technique 
+ * @name Arbitrary file write during zip extraction ("ZipSlip")
+ * @description Extracting files from a malicious zip archive without validating that the
+ *              destination file path is within the destination directory can cause files outside
+ *              the destination directory to be overwritten, due to the possible presence of
```

Comment:
Maybe truncate the description after "overwritten"? The rest of the information is duplicated in the query help file anyway.

Reply:
Done.